packages/react-components/src/types/VideoGalleryParticipant.ts

```diff
@@ -90,4 +90,10 @@ export interface VideoGalleryRemoteParticipant extends VideoGalleryParticipant {
   isSpeaking?: boolean;
   /** Video stream of shared screen */
   screenShareStream?: VideoGalleryStream;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /** State that the participant is in */
+  state?: string;
```

Comment:
Avoid `string` as the type here, contoso won't know how to use it, can we use a string literal instead?
e.g. `state: 'connected' | 'onHold' | etc..
The literal values should be only what we use inside our react-components code and not every possible option

Reply:
would something appropriate to do would maybe be to link the comment to the type our samples and composites would use? something @anjulgarg and I discussed was keeping the ACS headless types out of the components. If we restrict the string here to be the ones from calling we run into this issue here.

Reply:
Don't restrict it to the ones from the calling SDK, but instead restrict it to ones that are used inside our react-components codebase. What are we currently using this state for? What are the different possible values it could be with respect to the video gallery?